data/blog/battery-life.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Battery life instrumentation'
+date: '2023-04-25'
+tags:
+  - 'oss'
+  - 'mobile'
+authors: ['mihai']
+images: ['/images/battery-life/social.png']
+summary: 'Instrumentation that we’re adding to the Tailscale clients to have a scalable approach to battery life investigations.'
+---
+
+Battery consumption is one of the top areas where we’ve head that Tailscale needs to improve, via both [GitHub](https://github.com/tailscale/tailscale/issues/3363) and [social media](https://twitter.com/search?q=tailscale%20battery&src=typed_query&f=live). There’s always been [work on battery-related issues](https://github.com/tailscale/tailscale/issues?q=label%3Abattery+is%3Aclosed), but it’s not something that can ever be declared “done”:
+
+- Tailscale’s feature set evolves — using the [NextDNS integration](https://tailscale.com/blog/nextdns/) also means that Tailscale is now responsible for all DNS requests issued by a device.
+- The operating system changes, and Apple or Google may intentionally or [unintentionally](https://github.com/tailscale/tailscale/issues/4646#issuecomment-1257251332) change how the application operates and when it’s allowed to run, and how much power it consumes.
+- New patterns of user behavior may arise, for example there may new reasons to [use an exit node](https://www.youtube.com/watch?v=9CunwUs08og) more frequently.
```

Comment:
Agreed.

Reply:
(sorry, accidentally deleted comment so this is missing context now.  Was just saying I'm not sure we necessarily want to link to this)